components/common-go/baseserver/testing.go

```diff
@@ -46,20 +45,7 @@ func MustUseRandomLocalAddress(t *testing.T) *ServerConfiguration {
 func MustFindFreePort(t *testing.T) int {
 	t.Helper()
 
-	addr, err := net.ResolveTCPAddr("tcp", "localhost:0")
```

Comment:
Would be clearer to remove this function (as well as `MustUseRandomLocalAddress`) and directly use `localhost:0` on the few call sites we have

Reply:
Yep, if I can get the build passing on this change then we can clean it up and adjust.